libs/cardano-ledger-core/src/Cardano/Ledger/Plutus/ExUnits.hs

```diff
@@ -108,14 +110,14 @@ instance ToJSON ExUnits where
   toJSON exUnits@(ExUnits _ _) =
     let ExUnits {exUnitsMem, exUnitsSteps} = exUnits
      in object
-          [ "exUnitsMem" .= toJSON exUnitsMem
-          , "exUnitsSteps" .= toJSON exUnitsSteps
+          [ "memory" .= toJSON exUnitsMem
+          , "steps" .= toJSON exUnitsSteps
           ]
 
 instance FromJSON ExUnits where
   parseJSON = withObject "exUnits" $ \o -> do
-    exUnitsMem <- checkWord64Bounds =<< o .: "exUnitsMem"
-    exUnitsSteps <- checkWord64Bounds =<< o .: "exUnitsSteps"
+    exUnitsMem <- checkWord64Bounds =<< o .: "memory"
+    exUnitsSteps <- checkWord64Bounds =<< o .: "steps"
```

Comment:
We'll need to accept previous keys as well, since some types like `ExUnits` are also present in the genesis file, which we can't change.
Something like this could do:
```suggestion
    exUnitsMem <- checkWord64Bounds =<< (o .: "memory" <|> o .: "exUnitsMem")
    exUnitsSteps <- checkWord64Bounds =<< (o .: "steps" <|> o .: "exUnitsSteps")
```

Alternatively, we need to use a different parser for genesis FromJSON instances. Which could be safer, I think, but I don't have a strong opinion on it, since I don't know how JSON instances for PParams are used